src/main/java/com/assignments/questions/Reservation.java

```diff
@@ -0,0 +1,27 @@
+package com.assignments.questions;
+
+// Reservation Class - Question 2;
+public class Reservation {
+    String ticketID;
+
+    public Reservation(String ticketID){
+        this.ticketID = ticketID;
+    }
+
+    public String getTicketID(){
```

Comment:
You don't need the getter and setter for this question. The constructor is enough